index.js

```diff
@@ -28,101 +24,90 @@ function parseLocale(x) {
 }
 
 function getLocale(str) {
-	return (str && str.replace(/[.:].*/, '')) || fallback();
+	return (str && str.replace(/["]?(.*)[.:].*/, '$1')) || fallback();
```

Comment:
I'm sorry but I don't know what do you mean by regression test in this scenario. Could you care to explain, please?